app/views/coronavirus/edit_page.html.erb

```diff
@@ -0,0 +1,24 @@
+<%
+  links = [
+    {
+      text: 'Coronavirus topic pages',
+      href: coronavirus_index_path
+    },
+    {
+      text: "Coronavirus (COVID-19)"
+    }
+  ]
+
+  metadata = {
+    "Status" => "Draft",
+    "Last saved" => format_full_date_and_time(DateTime.now),
+  }
+%>
+
+<% content_for :breadcrumbs, render('shared/steps/step_breadcrumb', links: links) %>
+<% content_for :title, "Coronavirus (COVID-19)" %>
+<% content_for :context, 'Edit guidance and support accordion' %>
+
+<div class="covid-manage-page govuk-grid-row">
```

Comment:
Not sure if it's just me but looks like the alignment is a bit off here?

Reply:
Yep you are correct – this should have been resolved now